src/bin/cargo-hfuzz.rs

```diff
@@ -11,6 +11,77 @@ const HONGGFUZZ_WORKSPACE: &str = "hfuzz_workspace";
 #[cfg(target_family="windows")]
 compile_error!("honggfuzz-rs does not currently support Windows but works well under WSL (Windows Subsystem for Linux)");
 
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+#[structopt(name = "cargo-hfuzz", about = "Fuzz your Rust code with Google-developed Honggfuzz !")]
+struct Opt {
+    #[structopt(subcommand)]
+    command: OptSub,
+}
+
+#[derive(Debug, StructOpt)]
+struct CommonOpts {
+    /// only build binary but don't execute it
+    #[structopt(long)]
+    only_build: bool,
+
+    /// flags given to `rustc`, for example "-Z sanitizer=address"
+    #[structopt(long, env = "RUSTFLAGS")]
+    rustflags: Option<String>,
+
+    /// args given to `cargo build`
+    #[structopt(long, env = "HFUZZ_BUILD_ARGS")]
+    build_args: Option<String>,
+
+    /// path to working directory
+    #[structopt(short, long, default_value = "hfuzz_workspace", env = "HFUZZ_WORKSPACE")]
+    workspace: String,
+}
+
+#[derive(Debug, StructOpt)]
+enum OptSub {
+    /// build and run fuzzing
+    Fuzz {
+        #[structopt(flatten)]
+        common_opts: CommonOpts,
+
+        /// path to fuzzer's input files (aka "corpus"), relative to `$HFUZZ_WORKSPACE/{TARGET}`
+        #[structopt(short, long, default_value = "input", env = "HFUZZ_INPUT")]
+        input: String,
+
+        /// which binary target to fuzz
+        target: String,
```

Comment:
I'd be in favour of changing this to `--bin`, since target is rather ambiguous with `cargo` having `--target <TRIPLE>... Build for the target triple`.

Reply:
ah yes, `target` is indeed confusing.
however, as of now, this is not an optional argument (a la `--bin`), but we could rename it as `binary` and it would be reflected in the help messages.
WDYT?